apps/judicial-system/web/src/components/InfoCard/InfoCardActiveIndictment.tsx

```diff
@@ -41,8 +41,11 @@ const InfoCardActiveIndictment: React.FC = () => {
         {
           title: formatMessage(m.offence),
           value: capitalize(
-            workingCase.indictmentSubType
-              ? indictmentSubTypes[workingCase.indictmentSubType]
+            workingCase.indictmentSubTypes &&
+              Object.entries(workingCase.indictmentSubTypes).length > 0
+              ? indictmentSubTypes[
+                  Object.entries(workingCase.indictmentSubTypes)[0][1][0]
```

Comment:
This is unclear to me. Please consider writing a helper function for this and explain the `[0][1][0]` thing with a comment

Reply:
This is throwaway code, so I would rather not do that. The whole logic will change when the client implements multiple indictment sub-types.